jbpm-flow/src/main/java/org/jbpm/workflow/instance/node/AsyncEventNodeInstance.java

```diff
@@ -37,7 +37,9 @@
  *
  */
 public class AsyncEventNodeInstance extends EventNodeInstance {
-    
+
+    public static final String METADATA_WRAP = "wrap";
```

Comment:
this should be moved to node instead of node instance, and would actually keep it on generic node level rather than async

and maybe give it a bit more meaningful name